app/sentry.ts

```diff
@@ -45,6 +45,16 @@ if (!config.sentry.dsn.includes('dummy')) {
                     adapter_errors_json: JSON.stringify(error.errors)
                 }
             }
+
+            try {
+                // Try to store JSON of error for diagnosing bugs
+                event.extra = {
+                    ...event.extra,
+                    error_json: JSON.stringify(error)
```

Comment:
Codacy found an issue: [Expected indentation of 10 spaces but found 20.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6007450)